### ISSUE TITLE
Alt key shortcuts not available anymore in the Instance Editing Panel

### ISSUE THREAD

**Issue by helmo2019** [PrismLauncher/PrismLauncher#2523]
### Operating System
Debian 12

### Version of Prism Launcher

Prism Launcher 8.3 (Flatpak)

### Version of Qt

Qt 6.7.1

### Description of bug

In all pages of the Instance Editing Window, Alt-key-shortcuts are not available anymore for the right-hand buttons ("Change Version" or "Download Mods", for instance). I definitely remember using being Alt-key-shortcuts for these buttons in an older version, indicated by an underlined letter in the button labels.

### Steps to reproduce

1. Edit an instance
2. Observe there being no underlined letters in any of the right-hand buttons in any page/panel

### Suspected cause

In a support channel, I was told that there were no obvious issues with the code regarding this problem, and that it may be related to Qt.

### This issue is unique

- [x] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
You remember wrong: this is not a bug but a feature request

**Comment by helmo2019** [comment]
No, I 100% remember using Alt shortcuts to enable, disable, and update my mods. But I guess it may as well leave it labeled as feature request.
